packages/clarity-decode/package.json

```diff
@@ -26,24 +26,24 @@
     "url": "https://github.com/Microsoft/clarity/issues"
   },
   "dependencies": {
-    "clarity-js": "^0.7.2"
+    "clarity-js": "^0.7.3"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^19.0.1",
-    "@rollup/plugin-node-resolve": "^13.0.2",
-    "del-cli": "^4.0.1",
-    "husky": "^7.0.1",
-    "lint-staged": "^11.0.1",
-    "rollup": "^2.53.2",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.30.0",
+    "@rollup/plugin-commonjs": "^24.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.0",
+    "@rollup/plugin-terser": "^0.4.0",
+    "@rollup/plugin-typescript": "^11.0.0",
+    "del-cli": "^5.0.0",
+    "husky": "^8.0.0",
+    "lint-staged": "^13.1.0",
+    "rollup": "^3.0.0",
     "ts-node": "^10.1.0",
     "tslint": "^6.1.3",
     "typescript": "^4.3.5"
```

Comment:
This is pretty old. [https://github.com/microsoft/TypeScript/releases/tag/v4.3.5](https://github.com/microsoft/TypeScript/releases/tag/v4.3.5). From Jan 2021. 
Should we upgrade to current (4.9.5)?

Reply:
Good catch, I will address this in a subsequent change.